### PR TITLE
SJRK-190

### PR DESCRIPTION
**Pull request by BlueSlug**
This pull request makes changes to address the design update in SJRK-190, as described here:
- https://issues.fluidproject.org/browse/SJRK-190

It also makes a few styling fixes and incorporates the changes in the SJRK-174 PR as noted above.

### CODE REVIEW

File: src/messages/storyMessages_es.json [hunk 1]
```diff
@@ -33,6 +33,8 @@
     "message_menu_storyBuilderText": "Constructor de historias",
     "message_menu_english": "English",
     "message_menu_spanish": "Español",
+    "message_important_notes_title": "Notas importantes",
+    "message_important_notes_body": "<li>Su historia publicada será licenciada bajo CC BY 4.0.</li><li>No podrás realizar cambios en tu historia una vez que la publiques.</li><li>Puede eliminar su historia enviando un correo electrónico a stories@idrc.ocadu.ca</li>",
```
Comment (simonbates) [inline src/messages/storyMessages_es.json]: The Spanish version isn't rendered properly. I see the raw HTML tags. On Firefox.

File: src/messages/storyMessages_en.json [hunk 2]
```diff
@@ -38,6 +38,8 @@
     "message_menu_storyBuilderText": "Story Builder",
     "message_menu_english": "English",
     "message_menu_spanish": "Español",
+    "message_important_notes_title": "Important notes",
+    "message_important_notes_body": "* Your published story will be licensed under [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/).\n* You won't be able to make changes to your story once you publish.\n* You can have your story removed by emailing [stories@idrc.ocadu.ca](mailto:stories@idrc.ocadu.ca)",
```
Comment (simonbates) [inline src/messages/storyMessages_en.json]: I think there is some risk in linking to the CC license here -- a user can easily lose their story:

- Write story
- Preview it
- See the Important Notes, click on the "CC BY 4.0" link, page opens, replacing the story editor
- Click back to return to the story editor
- Story is gone

Some options:

- Remove the link, let users find it on their own
- Open the link in a new tab: https://stackoverflow.com/a/15551842

File: src/css/storyTelling.css [hunk 3]
```diff
@@ -107,28 +107,8 @@ fieldset ul {
     border-color: #CCC;
 }
 
-.sjrk-st-button-text-block-container {
-    grid-column: 1;
-}
```
Comment (simonbates) [inline src/css/storyTelling.css:112]: The removal of these CSS classes is on purpose? Does it have to do with SJRK-190, or just unrelated clean-up?

File: src/messages/storyMessages_en.json (hunk 2 above)
Comment (simonbates) [reply src/messages/storyMessages_en.json]: How is the user experience in iOS when opening in a new tab?

File: src/css/storyTelling.css (hunk 3 above)
Comment (BlueSlug) [reply src/css/storyTelling.css:112]: This was unrelated cleanup, yes. In hindsight, I should have made it under a NOJIRA commit at the very least.

File: src/messages/storyMessages_es.json (hunk 1 above)
Comment (BlueSlug) [reply src/messages/storyMessages_es.json]: This was an oversight, as I had originally stored the formatting as HTML and then switched to Markdown.

File: src/messages/storyMessages_en.json (hunk 2 above)
Comment (BlueSlug) [reply src/messages/storyMessages_en.json]: I have removed the links, as I agree with your assessment. There are links to the licence information and the email address in the footer, so hopefully the user would find them there.

File: src/css/storyTelling.css (hunk 3 above)
Comment (BlueSlug) [reply src/css/storyTelling.css:112]: In fact, more accurately, it's cleanup after fixing a styling bug related to the button positioning that only manifested itself on smaller screens.
